### PR TITLE
Bugfix/rpnv1 75 unreliable messagecentralimpltest

### PR DESCRIPTION
**Pull request by clstrfsck**
Additional fixed for broken tests etc.

### CODE REVIEW

File: radixdlt/src/test/java/org/radix/network2/messaging/MessageCentralImplTest.java [hunk 1]
```diff
@@ -80,26 +105,29 @@ public void testSetup() {
 
 		// Curse you singletons
 		Universe universe = mock(Universe.class);
-		when(universe.getMagic()).thenReturn(0);
+		doReturn(0).when(universe).getMagic();
```
Comment (Sajjon) [inline radixdlt/src/test/java/org/radix/network2/messaging/MessageCentralImplTest.java:108]: what is the difference between `when(universe.getMagic()).thenReturn(0)`  and `doReturn(0).when(universe).getMagic();` ?

**Review by Sajjon — APPROVED** [review]
LGTM

File: radixdlt/src/integration/java/com/radixdlt/consensus/CrashFaultNetworkTest.java [hunk 2]
```diff
@@ -204,24 +205,20 @@ public void given_7_correct_bfts_that_randomly_crash__then_correct_instances_sho
...
-				allNodes.stream()
-					.map(node -> bftNetwork.getVertexStore(node).lastCommittedVertex()
-						.map(vertex -> Pair.of(node, vertex)))
-					.collect(Collectors.toList()),
-				Arrays::stream)
+			allNodes.stream()
+				.map(node -> bftNetwork.getVertexStore(node).lastCommittedVertex()
+					.map(vertex -> Pair.of(node, vertex)))
+				.collect(Collectors.toList()),
+				this::toStream)
```
Comment (flotothemoon) [inline radixdlt/src/integration/java/com/radixdlt/consensus/CrashFaultNetworkTest.java:213]: Why this change from `Array::stream` to `this::toStream`

**Review by flotothemoon — APPROVED** [review]

**Review by talekhinezh — APPROVED** [review]

File: radixdlt/src/test/java/org/radix/network2/messaging/MessageCentralImplTest.java (hunk 1 above)
Comment (clstrfsck) [reply radixdlt/src/test/java/org/radix/network2/messaging/MessageCentralImplTest.java:108]: It really has to do with the complicated and static way that Mockito uses to set up mocking statically with spy() methods.  See here for more details: http://sangsoonam.github.io/2019/02/04/mockito-doreturn-vs-thenreturn.html
